app/controllers/admin/schools/induction_coordinators_controller.rb

```diff
@@ -31,8 +54,36 @@ def set_school
       @school = School.find params[:school_id]
     end
 
-    def form_params
-      params.require(:nominate_induction_tutor_form).permit(:full_name, :email)
+    def school_induction_tutor
+      @school.induction_coordinators.first
+    end
```

Comment:
I wonder if this is something we would like on `school` model - so that we don't duplicate a logic of 'who is the induction tutor for a school'. And so that eventually it's easier to change it, if we choose to.

Reply:
Yes, I thought the same but didn't want to do more changes! I'll make that change though as it will help